### PR TITLE
Tic Tac Toe Game in Ruby

### PR DESCRIPTION
**Pull request by Tahirbhalli**
Ruby
collaborative work
Object-oriented programing

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# add the linters you want stickler to use for this project
+linters:
+  rubocop:
+    display_cop_names: true
+    # indicate where is the config file for stylelint
+    config: "./rubocop.yml"
+    fixer: true
```
Comment (MauricioRobayo) [inline .stickler.yml]: Please do not modify the provided files: https://github.com/microverseinc/linters-config/tree/master/ruby.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
This is awesome, you have made a wonderful job!

But we need to fix some issues before continuing to fit the requirements:

* [x] Do not modify `.stickler.yml` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).
* [x] You should have a `lib` __directory__ (with .gitkeep file OR any other **empty** file(s)). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).
* [x] You should have a `bin` __directory__ with `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/bin_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).
* [x] `main.rb` is should be an [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) with `#!/usr/bin/env ruby` at the top ([shebang](https://en.wikipedia.org/wiki/Shebang_(Unix))).
* [x] `main.rb` should put "Hello world" when executed (https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).

This should be the structure of the project:

```
├── bin
│   └── main.rb  => executable file with shebang at top
└── lib
    └── .gitkeep
```

You can save the progress that you already made on a new branch, and adjust this one to fit the requirements.

Status: __Request changes__.

**Review by MauricioRobayo — APPROVED** [review]
Well done!

The project looks great and all requirements are met.

Status: __Approved__.

**Review by Forison — COMMENTED** [review]
Hi @Tahirbhalli, The last review suggests the project has been **approved** hence I can not review the project. Check your dashboard to see if the milestone has been marked as completed else let me know so I can send a report.🙂

**Pull request closed without merging**